templates/guides/import-script.md

```diff
@@ -0,0 +1,132 @@
+```
+3. Run the import script. Insert the script's version, your computer's operating system, and your computer's architecture for `<version-number>`, `<os>` and `<arc>`.
+
+- On Unix-based systems:
+```
+./terraform-provider-astro-import-script_&lt;version-number&gt;_&lt;os&gt;_&lt;arc&gt; [options]
+```
+- On Windows:
+
+```
+.\terraform-provider-astro-import-script_&lt;version-number&gt;_&lt;os&gt;_&lt;arc&gt;.exe [options]
+```
+
+To import your existing Workspace, API token and Team, specify those resources with the `-resources` option. The other option you will need to specify is `-organizationId`:
+```
+./terraform-provider-astro-import-script_v0.1.3_darwin_arm64 -organizationId &lt;your-organization-id&gt; -resources api_token,team,workspace
+```
+
+You should see the following output:
+```
+Terraform Import Script Starting
+Resources to import:  [api_token team workspace]
+Using organization ID: &lt;your-organization-id&gt
+Terraform version 1.9.7 is installed and meets the minimum required version.
+Importing teams for organization &lt;your-organization-id&gt
+Importing API tokens for organization &lt;your-organization-id&gt
+Importing workspaces for organization &lt;your-organization-id&gt
+Importing Workspaces: [&lt;workspace-id&gt]
+Successfully handled resource workspace
+Importing API Tokens: [&lt;api_token-id&gt]
+Successfully handled resource api_token
+Importing Teams: [&lt;team-id&gt]
+Successfully handled resource team
+Successfully wrote import configuration to import.tf
+Successfully deleted generated.tf
+terraform.tfstate does not exist, no need to delete
+astro_team.team_&lt;team-id&gt: Preparing import... [id=&lt;team-id&gt]
+astro_api_token.api_token_cm2c9slvb08p601n3lb4jljps: Preparing import... [id=cm2c9slvb08p601n3lb4jljps]
+astro_workspace.workspace_&lt;workspace-id&gt: Preparing import... [id=&lt;workspace-id&gt]
+astro_workspace.workspace_&lt;workspace-id&gt: Refreshing state... [id=&lt;workspace-id&gt]
+astro_api_token.api_token_&lt;api_token-id&gt: Refreshing state... [id=&lt;api_token-id&gt]
+astro_team.team_&lt;team-id&gt: Refreshing state... [id=&lt;team-id&gt]
+
+Terraform will perform the following actions:
+
+...
+
+Plan: 3 to import, 0 to add, 0 to change, 0 to destroy.
+
+Terraform has generated configuration and written it to generated.tf. Please review the configuration and edit it as
+necessary before adding it to version control.
+
+Note: You didn't use the -out option to save this plan, so Terraform can't guarantee to take exactly these actions if you
+run "terraform apply" now.
+Import process completed. Summary:
+Resource workspace processed successfully
+Resource api_token processed successfully
+Resource team processed successfully
+```
+-> If you import Deployments, they will not count torwards the `Plan: 3 to import, 0 to add, 0 to change, 0 to destroy` line of the output even when the Deployments are successfully imported.
```

Comment:
add a note that this is a known issue right now and will be fixed in the future?